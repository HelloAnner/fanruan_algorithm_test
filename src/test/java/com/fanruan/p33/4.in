5
49114
55605
56857
64703
99863
4
712001
1341177
2320326
7627675
