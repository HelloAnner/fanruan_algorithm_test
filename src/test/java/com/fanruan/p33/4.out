cd571095bb59f5c0c0fbb4532e756ac8